### DISCUSSION TITLE
Issue with applying to GitHub student developer pack

### DISCUSSION THREAD

**Discussion by adel051700** [community/community#102772] [GitHub Education] [answered]
### Hi everyone,

When I try to press process application, it doesn’t do anything, I’ve waited 5-10 minutes and nothing happened. Is there anything I can do?

**Comment by Gapy05** [comment]
same

**Comment by sachasa** [comment]
I had the same issue

**Comment by chavachai** [comment]
Hi, I'm not quite sure but did you try troubleshooting with the following steps?

1. Refresh the page.
2. Check for errors or warnings.
3. Inspect the browser console for clues.
4. Clear browser cache and cookies.
5. Try a different browser.

If nothing works Contact support for assistance or elaborate the root cause for clear insights of the issue.

**Reply by sachasa** [reply]
Thanks 👍

**Comment by Gapy05** [comment, marked as answer]
It worked when i applied on my phone and took the picture of the ID directly(not from album).

**Reply by adel051700** [reply]
Thank you it worked

**Reply by mahmudhasan-k** [reply]
I tried the same. But it doesnt change to back camera. And im having hard time focusing from front camera.....
